pkg/client/unversioned/testclient/fake_templates.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testclient
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/watch"
+)
+
+// FakeTemplates implements TemplatesInterface. Meant to be embedded into a struct to get a default
+// FakeTemplates. This makes faking out just the methods you want to test easier.
+type FakeTemplates struct {
+	Fake      *FakeExperimental
+	Namespace string
+}
+
```

Comment:
Add a static check to make sure FakeTemplates implements TemplatesInterface here too